WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -65,6 +72,25 @@ private void ValidateBitcoinP2PEndPoint(IValidationErrors errors)
 		}
 	}
 
+	private void ValidateCoordinatorUri(IValidationErrors errors)
+	{
+		Regex uriRegex = new Regex(@".*", RegexOptions.Compiled);
```

Comment:
@turbolay Could you help me out with a regex here?

Reply:
Please don't. Validate uris using the Uri.TryCreate method. Remember that not all uris are valid for us, only those which scheme is http/https

Reply:
Done https://github.com/zkSNACKs/WalletWasabi/pull/13090/commits/4d4266f6bca3a2c3870cc76879db8425fc8dcefa